tests/parser/Parser.test.js

```diff
@@ -1743,5 +1743,19 @@ describe('Parser', () => {
         expect(parsed[2]).toBe(0);
       });
     });
+    describe('Conditional Operator', () => {
+      test('roll `1?5:0', () => {
```

Comment:
Looks good. Could you also add tests to check against ternaries with other value types (ie. Dice, math formulas etc), in condition, and the true / false expressions?